services/validation/internal/service/validation_service.go

```diff
@@ -68,6 +68,36 @@ func (svc *validationService) ValidateNode(node *node.Node) {
 	})
 }
 
+func (svc *validationService) readFromProfileURL(profileURL string) (interface{}, error) {
+	document := gojsonschema.NewReferenceLoader(profileURL)
+
+	var data interface{}
+
+	// TODO: Need a feature toggle mechanism.
+	if true {
```

Comment:
@geoffturk Set to `false` to test exponential backoff.

We probably need a feature toggle mechanism to turn on/off this feature whenever we want.

Reply:
OK. I made a note about it in the feature toggle issue.